ts-client/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "types": ["jest", "mocha", "chai"],
     "typeRoots": ["./node_modules/@types"],
     "module": "commonjs",
-    "target": "es6",
+    // "target": "es6",
```

Comment:
Why this was changed? Does this impact anything?

Reply:
this is required to work for bigint